src/components/AppNavigationCaption/AppNavigationCaption.vue

```diff
@@ -1,6 +1,7 @@
 <template>
-	<li class="app-navigation-caption--item">
-		{{ text }}
+	<li class="app-navigation-caption"
+		@click="onClick">
```

Comment:
This doesn't work for me. Do we need a `<a>` tag?

Reply:
Wait a second, `click.native` doesn't work from the parent?
According to the vuejs docs we should not need to emit it back like that on such simple component :(

Reply:
> This doesn't work for me. Do we need a <a> tag?

I've been trying to do this yesterday and I haven't found  a way to make it work.. Tried the `<a>` as well..

Reply:
Fixed!
We indeed don't need to forward the click with an internal event, and if we leave the css `pointer-events: none` we'll definitely won't receive anything :grin: :see_no_evil: